### ISSUE TITLE
Cannot install requirements

### ISSUE THREAD

**Issue by akirayorunoe** [pijuskri/bilibili-caption-project#2]
Hi, i cannot install the requirements.txt 
```
pip install -r requirements.txt
Looking in indexes: https://download.pytorch.org/whl/cu117
Looking in links: https://www.paddlepaddle.org.cn/whl/windows/mkl/avx/stable.html
ERROR: Could not find a version that satisfies the requirement paddlepaddle-gpu==2.4.2.post117 (from versions: none)
ERROR: No matching distribution found for paddlepaddle-gpu==2.4.2.post117
```

**Comment by AntoniosBarotsis** [comment]
Not sure but it could be that you are using an [unsupported Python version](https://pypi.org/project/paddlepaddle-gpu/)?

If you are in 3.11 try using one of the versions listed there perhaps and it should be fine.

**Comment by pijuskri** [comment]
Yep, just installed with empty env again and it does work with python 3.9. Paddle has not been that well updated but it is the best tool ive found so i think sticking to versions below 3.10 is needed.

**Comment by akirayorunoe** [comment]
<img width="915" alt="Screen Shot 2024-01-05 at 10 09 00" src="https://github.com/pijuskri/bilibili-caption-project/assets/51730524/963b89f4-bd38-4641-a79c-5f9679df8b95">
Sorry for late reply, i have downgrade to python 3.9 but still have the same issue

**Comment by AntoniosBarotsis** [comment]
Are you on Linux or WSL?

> - CentOS 7 (GPUVersion Supports CUDA 10.2/11.2/11.6/11.7)
> - Ubuntu 16.04/18.04/20.04/22.04 (GPUVersion Supports CUDA 10.2/11.2/11.6/11.7)

Are the *required* versions (including CUDA). 

If that doesn't work then I'm not sure :/

**Comment by pijuskri** [comment]
Can you also perhaps try installing the dependencies with pip directly? I had some issues where, using the requirements file, pip was not be able to find the package, but "pip install <package>" worked just fine.

**Comment by akirayorunoe** [comment]
@pijuskri so pip install each dependencies in requirement file individually?

**Comment by pijuskri** [comment]
Yeah, start with paddleocr to see if that fixes yhe issue
